src/redux/conference-details.js

```diff
@@ -0,0 +1,80 @@
+var keyMirror = require('keymirror');
+var api = require('../mixins/api.jsx').api;
+
+var Types = keyMirror({
+    SET_DETAILS: null,
+    SET_DETAILS_FETCHING: null,
+    SET_DETAILS_ERROR: null
+});
+
+module.exports.detailsReducer = function (state, action) {
+    if (typeof state === 'undefined') {
+        state = {};
+    }
+    switch (action.type) {
+    case Types.SET_DETAILS:
+        return action.details;
+    case Types.SET_DETAILS_FETCHING:
+        return {fetching: action.fetching};
+    case Types.SET_DETAILS_ERROR:
+        return {error: action.error};
```

Comment:
This will replace everything in `this.props.details` with the details, `fetching`, or `error` – which is intentional, since that prop should not contain multiple of those states at any point in time.